TezosKitTests/GetBallotsRPCTest.swift

```diff
@@ -0,0 +1,14 @@
+// Copyright Keefer Taylor, 2018
+
+import TezosKit
+import XCTest
+
+class GetBallotsRPCTest: XCTestCase {
+  public func testGetBallotsRPC() {
+    let rpc = GetBallotsRPC { _, _ in }
+
+    XCTAssertEqual(rpc.endpoint, "chains/main/blocks/head/votes/ballots")
```

Comment:
This isn't a good thing to test, it just means you have twice the amount of places to update when changing the endpoint.  Test the parsing of the responses.